src/main/java/org/folio/invoices/utils/HelperUtils.java

```diff
@@ -68,7 +81,7 @@ private HelperUtils() {
 
   public static CompletableFuture<JsonObject> getVoucherById(String id, String lang, HttpClientInterface httpClient, Context ctx,
       Map<String, String> okapiHeaders, Logger logger) {
-    String endpoint = String.format(GET_VOUCHER_BY_ID, id, lang);
+    String endpoint = String.format(VOUCHER_BY_ID_ENDPOINT, id, lang);
```

Comment:
permission to get voucher by id is missing "voucher-storage.vouchers.item.get" in module descriptor

Reply:
This permission is not used when transitioning to "Approved". The getVouchers() method is used to obtain an existing voucher. 
Renamed this constant to make it more generic because it is used not only for the GET, but also for PUT.